docker/epoch_node1_mean16.yaml

```diff
@@ -7,6 +7,10 @@ http:
     external:
         peer_address: http://node1:3013/
 
+websocket:
+    internal:
+        listen_address: 0.0.0.0
```

Comment:
Shall the internal HTTP port be set too? Did you test that?

Reply:
It can be set, I didn't test it, because that functionality was there before my PR already.

Reply:
I expect [this](https://github.com/aeternity/epoch/pull/740/files?diff=split#diff-735feba3cdeb8646424e3f5fe388244dR66) changed things, I suggest retest of this point.

Reply:
```yaml
websocket:
    internal:
        listen_address: 0.0.0.0
        port: 4000```

This worked

Reply:
> Shall the internal HTTP port be set too? Did you test that?

@amir-hadi In my comment was not precise - I apologise. I think we were not on the same page.

I meant the internal HTTP *listen address* - not port. I.e. this:

> I expect [this](https://github.com/aeternity/epoch/pull/740/files?diff=split#diff-735feba3cdeb8646424e3f5fe388244dR66) changed things, I suggest retest of this point.

Reply:
I hope that I got you know, I did test, too.

```yaml
http:
    external:
        peer_address: http://node1:3013/
        listen_address: 0.0.0.0
    internal:
        listen_address: 0.0.0.0

websocket:
    internal:
        listen_address: 0.0.0.0
```

Reply:
Do I understand correctly that unless you amend docker/epoch_node1_mean16.yaml with the following section the user does not manage to contact internal HTTP endpoint?
```yaml
http:
    internal:
        listen_address: 0.0.0.0
```

(I check better doc and nginx config...)

Reply:
@amir-hadi What @lucafavatella is trying to explain that you also must change the ` docker/epoch_node1_mean16.yaml` to add:

```
http:
    internal:
        listen_address: 0.0.0.0
```

Because we changed the default listen address of the internal API from `0.0.0.0` to `127.0.0.1`, thus you cannot reach it anymore outside of the container, therefore the nginx proxy cannot reach it too as well.